gridcomps/ExtData2G/ExtDataYamlNodeStack.F90

```diff
@@ -0,0 +1,12 @@
+module MAPL_ExtDataYamlNodeStack
+   use yaFyaml
+   use MAPL_ExtDataYamlNodeWrapper
+   implicit none
+
+   public :: stack_depth
+   public :: yaml_node_stack
+
+   integer, save :: stack_depth = 0
+   type(YamlNodeWrapper), save :: yaml_node_stack(20)
```

Comment:
I assume this stack is "big enough" at 20? Want to make sure it's not a magic number we should document.

Reply:
It should be for now... hopefully @tclune can fix Yafyaml or at worst case I could try to rewrite this procedure without recursion